kolibri/core/assets/src/core-app/apiSpec.js

```diff
@@ -157,7 +157,7 @@ export default {
       PermissionsIcon,
       AppBarCorePage,
       CoreBase,
-      FullScreenSidePanel,
+      SidePanelModal,
```

Comment:
I probably asked this at the time, but why do we have this in the core API? I thought it was only used in learn?

Reply:
What a great question... will create an issue for this to follow-up on